scss/bitstyles/settings/shadows.stories.mdx

```diff
@@ -0,0 +1,145 @@
+
+## Generating shadows
+
+The simplest way to override the global shadow design tokens in `$bitstyles-shadows-shadows` is to use `shadow.generate()`:
+
+```scss
+@use '../tools/shadow';
+
+$bitstyles-shadows-shadows: (
+  'default': shadow.generate(
+    $color: rgba(palette.get('brand-1'), 0.15),
+    $offset: 2rem,
+    $blur-radius: 2rem,
+  ),
+);
+```
+
+This creates a specification for a multi-layered shadow using the `brand-1` color with a `2rem` blur, offset `2rem` downwards, which looks like the following examples.
+
+<Canvas>
+  <Story id="utilities-shadow--u-shadow-default" />
+  <Story id="utilities-drop-shadow--u-drop-shadow-default" />
+</Canvas>
+
+### Parameters
+
+|Name|Format|Description|
+|--|--|--|
+|$color|color|The color to be used. Probably you want a low-opacity version of a color. Avoid using pure black unless your design calls for it, as that can look flat, especially on non-white backgrounds.|
+|$offset|number|Vertical offset for the shadow. Include the units unless it’s zero. Optional, defaults to `0`.
+|$blur-radius|number|Maximum distance to blur the shadow. The extra layers in this multi-layer shadow will be smaller than the value you pass. Include the units.|
+|$layers|number|Number of layers of in the shadow. Optional, defaults to `4`.|
+
+## Using shadows
+
+The accessor function `shadows.get()` is the most straightfoward way to use shadows stored in this format. Specify the name of the shadow and the output format you need (these examples are taken from the settings files for `box-shadow` and `drop-shadow` utility classes, each of which uses a different format):
+
+```scss
+@use '../../tools/shadows';
+
+$bitstyles-box-shadow-values: (
+  'default': shadows.get('default', 'css'),
+);
+
+/*
+
+box-shadow:
+  0 0.5rem 0.5rem rgba(0, 13, 255, 0.15),
+  0 0.6666666667rem 0.6666666667rem rgba(0, 13, 255, 0.15),
+  0 1rem 1rem rgba(0, 13, 255, 0.15),
+  0 2rem 2rem rgba(0, 13, 255, 0.15);
+
+*/
+
+$bitstyles-drop-shadow-values: (
+  'default': shadows.get('default', 'filter'),
+);
+
+/*
+
+filter:
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 0.5rem 0.5rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 0.6666666667rem 0.6666666667rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 1rem 1rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 2rem 2rem);
+
+*/
+```
+
+### Using local shadows
+
+Not all shadows belong among the global shadows design tokens. Including all the shadows applied to buttons and input elements on hover/focus/active/disabled/etc. could swamp the global list. But you may want to use `shadow.generate()` to take advantage of the multi-layered shadows without having to hand-code them. You can do this using one of the two helper functions `shadow.to-css()` and `shadow.to-filter()`. These convert the list formatted-shadows provided by the generator into a format suitable for use in CSS `box-shadow` properties and CSS `filter` properties (using `drop-shadow`), respectively.
```

Comment:
Lacking the language skills, but this first sentence reads super weird to me 🙈 

Shouldn't this be
> not all shadows belong **to** the global...

Reply:
If that’s easier to parse, then yes :) I’ll update